internal/broadcaster/rate_broadcaster.go

```diff
@@ -186,15 +166,17 @@ utxoLoop:
 				continue
 			}
 			amount := utxo.Satoshis - fee
-			err = tx.PayTo(b.ks.Script, amount)
+			err = PayTo(tx, b.ks.Script, amount)
 			if err != nil {
 				return nil, fmt.Errorf("failed to pay transaction %d: %v", amount, err)
 			}
 
-			// Todo: Add OP_RETURN with text "ARC testing" so that WoC can tag it
+			err = tx.AddOpReturnOutput([]byte("ARC testing"))
+			if err != nil {
+				return nil, err
+			}
```

Comment:
Let's do this separately. There is a task for it
The text should be variable and it should be tested with WoC if it works as intended.

Reply:
Sure, I came back to TODO comment